main.cpp

```diff
@@ -127,393 +163,337 @@ struct CarWash
     int fretNote(int stringNum = 6, int fretNum = 0);
-    // clean fretboard
     void cleanFretboard();
-
 };
-/*
-Thing 7)  Body
-5 properties:
-    1) body type(std::string)
-    2) color (std::string)
-    3) cutout radius in cm (float)
-    4) number of cutouts (int)
-    5) thickness in cm (float)
-3 things it can do:
-    1) select pickup output
-    2) adjust pickup volume
-    3) adjust pickup tone
- */
+
+float Neck::adjustTrussRod(float numTurns)
+{
+    neckRelief *= numTurns * 0.1f;
+    return neckRelief;
+}
+
+int Neck::fretNote(int stringNum, int fretNum)
+{
+    return stringNum + fretNum;
+}
+
+void Neck::cleanFretboard() { fretboardSoiled = false;}
+
 struct Body
 {
-    // body type
     std::string bodyType = "solid";
-    // color 
     std::string color = "sunburst";
-    // cutout radius in cm 
     float cutoutRadius = 2.5f;
-    // number of cutouts 
     int numCutouts = 2;
-    // thickness in cm 
     float thickness = 4.76f;
 
-    // select pickup output
     int selectPickup(int pickup);
-    // adjust pickup volume
     float changeVolume(float newVolume);
-    // adjust pickup tone
     float changeTone(float newTone);
 };
-/*
-Thing 8) Tuners
-5 properties:
-    1) tuner metal (std::string)
-    2) string tension in pounds (double)
-    3) number of tuners(int)
-    4) tuner angle in degrees (double)
-    5) tuner gear ratio (float)
-3 things it can do:
-    1) rotate tuner key
-    2) adjust string tension
-    3) tighten set nut
- */ 
+
+int Body::selectPickup(int pickup) { return pickup;}
+
+float Body::changeVolume(float newVolume) { return newVolume;}
+
+float Body::changeTone(float newTone){ return newTone;}
+
 struct Tuners
 {
-    // tuner metal 
     std::string metal = "Steel";
-    // adjustment key orientation 
     double stringTension = 11.8;
-    // number of tuners
     int numTuners = 6;
-    // tuner angle in degrees
     double degree = 1.75;
-    // tuner gear ratio 
-    float gearRatio = 50.0;
+    double gearRatio = 50.0;
 
-    // rotate tuner key
-    void rotateKey(int keyNum = 1, float numTurns = 0.f);
-    // adjust string tension
+    void rotateKey(int keyNum = 1, double numTurns = 0.0);
     void changeStringTension(int keyNum, int tunedNote);
-    // get number of installed strings
     int getNumInstalledStrings();
     
 };
-/*
-Thing 9) Bridge
-5 properties:
-    1) type of bridge(std::string)
-    2) number of string poles (int)
-    3) length adjustment screw setting in mm (double)
-    4) type of tremolo (std::string)
-    5) floating bridge tightness setting (std::string)
-3 things it can do:
-    1) adjust string length
-    2) move whammy bar
-    3) adjust bridge height
- */
+
+void Tuners::rotateKey(int keyNum , double numTurns) 
+{
+    degree += keyNum*numTurns/gearRatio*360.0;
+}
+
+void Tuners::changeStringTension(int keyNum, int tunedNote)
+{
+    stringTension += keyNum*tunedNote*0.01;
+}
+
+int Tuners::getNumInstalledStrings() {return numTuners;}
```

Comment:
If you write single-line functions, make sure you have space around the `{ }` so the code is easier to read
- [ ] revise